packages/react/src/ui/Select/components/SelectContent.tsx

```diff
@@ -121,7 +121,7 @@ const SelectContent = forwardRef<
           asList || virtualReady ? "" : "opacity-0"
         )}
         style={{
-          height: virtualizer.getTotalSize(),
+          height: virtualizer.getTotalSize() + 8, // Add 8px to the height to account for the padding
```

Comment:
Magic number.
Would you consider to set this in a constant with a proper name then we can remove the comment as well and doing the code more readable